### ISSUE TITLE
Frontend won't compile

### ISSUE THREAD

**Issue by mausch** [logspace-ai/langflow#91]
I run `docker compose up`
logs end up with:
```
langflow-frontend-1  | ERROR in ./src/App.tsx 16:0-53
langflow-frontend-1  | Module not found: Error: Can't resolve 'react-error-boundary' in '/home/node/app/src'
langflow-frontend-1  | 
langflow-frontend-1  | webpack compiled with 1 error and 1 warning
langflow-frontend-1  | ERROR in src/App.tsx:12:31
langflow-frontend-1  | TS2307: Cannot find module 'react-error-boundary' or its corresponding type declarations.
langflow-frontend-1  |     10 | import { locationContext } from "./contexts/locationContext";
langflow-frontend-1  |     11 | import TabsManagerComponent from "./pages/FlowPage/components/tabsManagerComponent";
langflow-frontend-1  |   > 12 | import { ErrorBoundary } from "react-error-boundary";
langflow-frontend-1  |        |                               ^^^^^^^^^^^^^^^^^^^^^^
langflow-frontend-1  |     13 | import CrashErrorComponent from "./components/CrashErrorComponent";
langflow-frontend-1  |     14 | import { TabsContext } from "./contexts/tabsContext";
langflow-frontend-1  |     15 |
```

Apparently this broke at some point after 0.0.45

**Comment by Galz648** [comment]
what might be the use case for you to use docker instread of as a python installation?

**Comment by ogabrielluiz** [comment]
Run ` make dev build=1` and it should work.

**Comment by carlorscoelho** [comment]
Hey, @mausch have you tried like @ogabrielluiz said? It should work.

**Comment by mausch** [comment]
Hi, it seems there's a misunderstanding around why I opened this issue, I should have been more clear.

I wanted to check out this project. I noticed a docker-compose.yml in the root, apparently for local dev, so I execute it to get something running. @Galz648 this is easier than setting up a Python env, also if it's been dockerised already it means it'll be less work for me to run it on Kubernetes for example.
I noticed the frontend build was crashing so I reported it here, since other people will likely attempt to do the same thing I did and maybe the maintainers didn't notice this issue. I'm not actually looking to do dev on this project for now.

I then looked for a workaround, first thing I tried was running it on Python, and that worked.
I also tried the previous release, which also works, so I updated the issue so maintainers will find it easier to pinpoint where things broke.
I also then noticed the `docker_example` directory.

Hope that gives some more context :slightly_smiling_face: 
BTW thanks for this project, looks great, looking forward to the implementation of memories and other components :rocket:
